language/method_spec.rb

```diff
@@ -1688,6 +1688,22 @@ def m(a, b = nil, c = nil, d, e: nil, **f)
     end
   end
 
+  ruby_version_is '2.7'...'3.0' do
+    context 'when passing an empty keyword splat to a method that does not accept keywords' do
+      evaluate <<-ruby do
+          def m(a); a; end
+        ruby
+        h = {}
+
+        -> do
+          suppress_warning do
+            m(**h).should == {}
+          end
+        end.should_not raise_error
```

Comment:
Checking for not raising exception is excessive to it's a common practice in RubySpec not to use it at all. If exceptions is raised a test will fail anyway.